packages/nimble-components/src/select/tests/select.stories.ts

```diff
@@ -1,4 +1,4 @@
-import { html, repeat } from '@microsoft/fast-element';
+import { html, repeat, when } from '@microsoft/fast-element';
```

Comment:
Is this text from the filterMode attribute still accurate?
>The act of filtering will use the hidden attribute on the options to remove and re-add them to the visible set. Thus, any client-provided hidden settings of the options will be overridden.

Also, not part of this PR but worth quickly fixing: I noticed we called it `filterMode` with the property naming convention but for storybook we generally prefer the attribute naming convention `filter-mode`.